reconcile/utils/openshift_resource.py

```diff
@@ -193,6 +197,14 @@ def api_version_mutation(val1, val2):
             return True
         return val1 == val2
 
+    @property
+    def last_applied_configuration(self):
```

Comment:
Where is the usage of this?

Reply:
This was used when I tested the same thing but using the last applied annotation instead of the hash. Removed.